src/main/java/com/app/katchup/Meeting/MeetingService.java

```diff
@@ -0,0 +1,21 @@
+package com.app.katchup.Meeting;
+
+import com.app.katchup.Meeting.model.Meeting;
+import com.app.katchup.MeetingResponse.model.Decision;
+import com.app.katchup.MeetingResponse.model.MeetingInboxResponse;
+import com.app.katchup.MeetingResponse.model.MeetingResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MeetingService {
+    @Autowired
+    MeetingRepository meetingRepository;
+    public void createMeeting(Meeting meeting){
+        if(meetingRepository.findIfHostHasDuplicateMeeting(meeting.getHost(),meeting.getStartDateTime(),meeting.getEndDateTime(),
```

Comment:
please break it to two lines. update the method here based on the repository

Reply:
please watch for alignment in line#15

Reply:
Return null if there exists a meeting already if not, go ahead and call save(), then get the same meeting to verify the insertion is successful and then return that object to the controller.